packages/tools/pluggable-widgets-tools/configs/prettier.base.json

```diff
@@ -6,7 +6,6 @@
   "singleQuote": false,
   "printWidth": 120,
   "bracketSpacing": true,
-  "bracketSameLine": false,
```

Comment:
This option throws an error for me

Reply:
You need to update your dependencies. New typescript conflicts with previous `jsxBracketSameLine` and requires it.

Reply:
reverted